notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>"Querying exoplanet data by name with exo.MAST"</p><p> </p><p>In the introduction, it would be good to explain to the user what some of the main differences are between exo.MAST and astroquery.MAST. If I'm not familiar with either, how would I know which one to go to first, depending on my science goals? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>